www-client/torbrowser-bin/torbrowser-bin-14.5.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Official Tor Browser"
+
+HOMEPAGE="https://www.torproject.org"
+
+TOR_SRC_BASE_URI="https://www.torproject.org/dist/torbrowser/"
+
+SRC_URI="${TOR_SRC_BASE_URI}/${PV}/tor-browser-linux-x86_64-${PV}.tar.xz"
+inherit desktop
```

Comment:
Please adhere to the usual ordering of variables/inherit, i.e. inherit goes under EAPI, and no blank lines between most of the variables either (DESCIRPTION/HOMEPAGE/SRC_URI go together, newline, then LICENSE/SLOT/KEYWORDS, then newline, then *DEPEND, then newline, then phases).